aptos-move/framework/aptos-framework/sources/staking_proxy.spec.move

```diff
@@ -22,6 +22,25 @@ spec aptos_framework::staking_proxy {
     spec set_vesting_contract_operator(owner: &signer, old_operator: address, new_operator: address) {
         // TODO: Can't verify `update_voter` in while loop.
         pragma aborts_if_is_partial;
+
+        let owner_address = signer::address_of(owner);
+        let vesting_contracts = global<vesting::AdminStore>(owner_address).vesting_contracts;
+        let post post_vesting_contracts = global<vesting::AdminStore>(owner_address).vesting_contracts;
+        ensures exists i in 0..len(vesting_contracts): (global<vesting::VestingContract>(vesting_contracts[i])).staking.operator == old_operator
```

Comment:
This property does not hold. Please make sure there is no inconsistency issue before submitting the PR.

Reply:
Here is the CI test failure: https://github.com/aptos-labs/aptos-core/actions/runs/6251224938/job/16971891917?pr=10136#step:4:13267